### PR TITLE
Fix #13509: adds the possibility of keyboard shortcuts for super/subscript

### LINKED ISSUE #13509: Shortcuts for Subscript and Superscript
We have Ctrl-B and Ctrl-I for bold and italics. That's good.
Please also add shortcuts for subscript and superscript of selected text.
I do this often on lyric text, and it's inconvenient to have to use the mouse on the toolbar at the bottom (subscript button) every time.

Originally reported in https://musescore.org/en/node/335417

### PR DESCRIPTION
**Pull request by mmeyn**
Resolves: #13509

You can now assign keyboard shortcuts to “Format text: superscript” and “Format text: subscript”.

Other than _italic_ and **bold** subscript and superscript cannot be combined so these cannot be simple toggles (so you could set subscript and superscript at the same time). It’s more like a “three-way toggle”; the following illustrates it (using SUB and SUP for keyboard shortcuts):
```
normal      –SUB→ subscript
normal      –SUP→ superscript
subscript   –SUB→ normal
subscript   –SUP→ superscript
superscript –SUB→ subscript
superscript –SUP→ normal
```
This is how it is done in LibreOffice.

This PR does NOT include default keyboard shortcuts (yet) because I would leave the decision on whether and which defaults should be chosen to some UI designer.

- [x] I signed the [CLA](https://musescore.org/en/cla)
- [X] The title of the PR describes the problem it addresses
- [X] Each commit's message describes its purpose and effects, and references the issue it resolves
- [X] If changes are extensive, there is a sequence of easily reviewable commits
- [X] The code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [X] There are no unnecessary changes
- [X] The code compiles and runs on my machine, preferably after each commit individually
- [ ] I created a unit test or vtest to verify the changes I made (if applicable)

### CODE REVIEW

**Review by cbjeukendrup — APPROVED** [review]

**Comment by cbjeukendrup** [comment]
I'll wait a bit before merging (and before asking people to test), since we're still waiting for a decision about a default shortcut.

**Comment by bkunda** [comment]
>  ...we're still waiting for a decision about a default shortcut.

Having read through the discussion in #13509, it seems clear that if we provide default shortcuts for super/subscript, we will make some people happy and others not. 
Given this really is a very niche requirement, I think it should be fine to provide no default. The user can then readily apply whatever shortcut they want to these commands in Preferences. 
You can consider this my decision 🙂

**Pull request merged**
